### PR TITLE
pwa thanks page

### PR DESCRIPTION
**Pull request by shuklabhay**
## Description
Created the react thanks page

## Motivation and Context
Eugene
[](https://github.com/the-blue-alliance/the-blue-alliance/issues/6168)

## How Has This Been Tested?
Local run

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change API specifications or require data migrations)

### CODE REVIEW

File: pwa/app/routes/thanks.tsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+        <div>
+          <h3 className="text-2xl">Sponsors</h3>
+          <p>Thank you to our generous sponsors for allowing our app to run.</p>
+          <h4 className="text-xl">Platinum</h4>
+          <ul>
+            <li>
+              <p className="pl-1">
+                <a href="https://www.andymark.com/" title="AndyMark">
+                  AndyMark
```
Comment (shuklabhay) [inline pwa/app/routes/thanks.tsx:44]: the only thing im unsure about is the list items dont have bullet points/indentation

File: pwa/app/routes/thanks.tsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+                <a href="https://github.com/twbs/bootstrap" title="Bootstrap">
+                  Bootstrap
+                </a>
+                &apos;s HTML CSS JS framework
+              </p>
+            </li>
+            <li className="pl-1">
+              <p>
+                <a href="https://react.dev/" title="React">
+                  React
```
Comment (tervay) [inline pwa/app/routes/thanks.tsx:89]: More specifically using [Remix](https://remix.run/) now

**Review by tervay — CHANGES_REQUESTED** [review]
Appreciate the PR! Just a couple minor changes

File: pwa/app/routes/thanks.tsx [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+            If you would like to support the site, please consider{' '}
+            <a href="/donate">making a donation.</a>
+          </p>
+        </div>
+
+        <div>
+          <h3 className="text-2xl">Contributors</h3>
+          <p>
+            Thanks to our code contributors for our{' '}
+            <a href="/https://github.com/the-blue-alliance/the-blue-alliance/graphs/contributors">
```
Comment (fangeugene) [inline pwa/app/routes/thanks.tsx:22]: Leading slash breaks this link

**Review by fangeugene — CHANGES_REQUESTED** [review]
Minor fixes

**Review by fangeugene — APPROVED** [review]
ty!

**Pull request merged**